src/CommonITILObject.php

```diff
@@ -6759,7 +6759,7 @@ public function getTimelineItems(array $options = [])
             ];
         }
 
-        if ($params['is_self_service']) {
+        if ($params['is_self_service'] || (!$params['expose_private'] &&  $params['bypass_rights'])) {
```

Comment:
This new condition is a bit hard to read.

I think the fix has to be made on previous condition:
```
- if (!$params['expose_private'] && !Session::haveRight("followup", ITILFollowup::SEEPRIVATE)) {
+if (!$params['expose_private'] || (!Session::haveRight("followup", ITILFollowup::SEEPRIVATE) && !$params['bypass_rights'])) { 
```

Reply:
it's work too